apps/src/applab/firebaseCounters.js

```diff
@@ -0,0 +1,124 @@
+/* global Applab */
+
+import Firebase from 'firebase';
+import { getDatabase } from './firebaseUtils';
+
+const TABLE_ROW_COUNT_LIMIT = 10;
+
+/**
+ * Map representing the maximum number of writes allowed over each time interval.
+ *
+ * @type {Object.<string, number>} Map from rate limit interval in seconds
+ *     to the maximum number of operations allowed during that interval.
+ */
+const RATE_LIMITS = {
+  15: 300,
+  60: 600
+};
```

Comment:
The other possible approach is that, since we are going to be generating `rules.json` and inserting it into the apps package already anyway, that we could also generate a `firebaseConstants.js` file at the same time. I think this sounds unnecessarily complicated since we know the appOptions approach will work, and there's no clear advantage to generating the JS file. Let me know if you disagree.